tests/src/tests/roact.spec.tsx

```diff
@@ -301,37 +284,21 @@ export = () => {
 			let ref: Frame | TextLabel | undefined;
 			let ref2: Frame | TextLabel | undefined;
 			const test = (
-				<frame>
-					{false ? (
-						<frame Ref={rbx => (ref = rbx)} />
-					) : (
-						<textlabel Ref={rbx => (ref = rbx)} />
-					)}
-				</frame>
+				<frame>{false ? <frame Ref={rbx => (ref = rbx)} /> : <textlabel Ref={rbx => (ref = rbx)} />}</frame>
```

Comment:
Does the parenthesising of the jsx element change whether it becomes a prereq statement? If so, this might slightly change the test case

Reply:
Not as far as I can tell. Try shift+alt+F and see how the output changes. The only difference is a set of parentheses which I don't think impact this test.

[Playground link](https://roblox-ts.com/playground/#code/JYWwDg9gTgLgBAJQgQwMbwGZQiOAiAASgCMAPGAZwHps0Y8BuAKCYBMBTVAG2Snbi7t4fDAC44ASQB2FGMimp2zVBBnx+AXjgAeLMhDsAfEwCQAbwzIuFfgH44AClMndUffwTsMGsyVJwNQ0cRALg-AEoAXzgqYxNwuHEnExcYdnIeYnYuRC8fPwCghxCtCOjY0yimbSo9A2MgA)